_extras/checkout.md

```diff
@@ -247,10 +247,11 @@ also email you to let you know, as this information is typically not shared duri
 after you have attended your teaching demonstration.
 
 ## 4. Tracking Your Progress
-After you complete this workshop, your application will be matched to your training record and a profile will be created in The
+After you complete this workshop, your application or profile-creation form will be matched to your training record and used to 
+create a profile that you can access in The
 Carpentries database, AMY. You can monitor your checkout progress by [logging on to our database, AMY][amy-login], using your GitHub ID.
-If you have any questions or did not provide a GitHub user name when registering for
-training, please email us at [instructor.training@carpentries.org](mailto: instructor.training@carpentries.org).
+If you have any questions or did not provide a GitHub user name in your form, please email us 
+at [instructor.training@carpentries.org](mailto: instructor.training@carpentries.org).
 
 Keeping [your profile][trainee-profile] up to date with a current email address and local airport helps us to keep in touch about teaching
```

Comment:
also, I don't see `trainee-profile` defined below?

Reply:
it's there I think I'm just bad at alphabetizing :)

Reply:
and i'm bad clicking to view the whole page in PR view.